recipes/pygmsh/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pygmsh" %}
+{% set version = "6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygmsh-{{ version }}.tar.gz
+  sha256: c07e3a7313b7f5658c1b8329cc9f1aadcbd81db1d962acaeb874dca357478876
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - gmsh <4
```

Comment:
You also need `python-gmsh` right?

Reply:
I don't believe so, pre v7 pygmsh didn't appear to use it, it just wrong straight GEO code

Reply:
cc @nschloe